psmc_check.py

```diff
@@ -27,6 +27,8 @@
 import Ska.engarchive.fetch_sci as fetch
 from Chandra.Time import DateTime
 import Chandra.Time
+from Quaternion import Quat
```

Comment:
Hi Gregg,

I think you and I talked about this before and determined it was coming from your mail application.

These emails were listed as coming from Jean for me.